packages/apps/workshop/stories/design-system/components/select/select-webcomponent.stories.js

```diff
@@ -295,3 +295,24 @@ export const Load = forModule(moduleName).createElement(
     },
   ),
 );
+
+export const PropertyValue = forModule(moduleName).createElement(
+  () => compileTemplate(
+    `
+      <oui-select
+        items="$ctrl.items"
+        model="$ctrl.model"
+        match="name"
+        value-property="code"
+        name="country"
+      ></oui-select>
+      <span>model value = {{ $ctrl.model }}</span>
+    `,
+    {
+      $ctrl: {
+        model: 'FR',
+        items: countries.map(({ country }) => country),
```

Comment:
**nitpick:** `items` array seems to be just an array of strings. How is the value property used in this test case?

Reply:
It's not an array of strings, countries is of an array of objects with a property `country` which is an object with a property code (string) and a property name (string).